TimberAPIExample/Examples/UIBuilderExample/UIBuilderFragmentExample.cs

```diff
@@ -70,19 +71,19 @@ public void ClearFragment()
 
         public void UpdateFragment()
         {
-            this._root.ToggleDisplayStyle(true);
+            _root.ToggleDisplayStyle(true);
         }
 
         private void OnFirstToggleValueChange(bool value)
         {
             // Do some action when toggle changed value
-            TimberAPIExamplePlugin.Log.LogFatal("New first toggle value:" + value);
```

Comment:
Must say that this is a good change. I was confused as hell when I tried the example the first time, and it suddenly logged a warning when pressing a button :D